test/t/localjumperror.rb

```diff
@@ -3,6 +3,13 @@
 
 assert('LocalJumpError', '15.2.25') do
   assert_equal Class, LocalJumpError.class
+  assert_raise LocalJumpError do
```

Comment:
I think ISO 11.3.5 defines when LocalJumpError should be raised.
15.2.25 defined which methods LocalJumpError should have.